docs/source/deployment/azure-functions.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Deploying with Azure Functions
```

Comment:
I might suggest naming this "Deploying Apollo Server to Azure Functions" and omitting the description (I bet we use this current pattern for all these deployment articles, but that feels incrementally better to me